### ISSUE TITLE
Different default Downloads folder for root user

### ISSUE THREAD

**Issue by darkdragon-001** [jaymoulin/docker-jdownloader#58]
Follow up on #47:

When run as root user (without option or with `-u root`), then the default _Downloads_ folder is `/root/Downloads`. I don't know how JDownloader determines the default download folder, but it would be nice if the root case would also default to `/opt/JDownloader/Downloads`.

**Comment by jaymoulin** [comment]
As users requested for `/root/Downloads` as default folder (#57), this issue would regress behaviour.

If you find this product useful, please consider donation to help me continue maintain and create great products ([![PayPal donation](https://github.com/jaymoulin/jaymoulin.github.io/raw/master/ppl.png "PayPal donation")](https://www.paypal.me/jaymoulin) [![Buy me a coffee](https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png "Buy me a coffee")](https://www.buymeacoffee.com/3Yu8ajd7W) [![Become a Patron](https://badgen.net/badge/become/a%20patron/F96854 "Become a Patron")](https://patreon.com/jaymoulin))

**Comment by darkdragon-001** [comment]
@jaymoulin The linked issue (#57) is not connected to the `/root/Downloads` issue. Therefore, I don't understand your answer.

**Comment by jaymoulin** [comment]
Typo, #47

**Comment by darkdragon-001** [comment]
JDownloader checks the environment variable `XDG_DOWNLOAD_DIR` ([GeneralSettings.java](https://github.com/mirror/jdownloader/blob/dae2fd29d9abcfd80d7f1985cf70d515c4eb0140/src/org/jdownloader/settings/GeneralSettings.java#L45), [CrossSystem.java](https://github.com/svn2github/appwork-utils/blob/41e2dedd169c41478f8f6057c49637af685a3efd/src/org/appwork/utils/os/CrossSystem.java#L370), [DesktopSupportLinux.java](https://github.com/svn2github/appwork-utils/blob/41e2dedd169c41478f8f6057c49637af685a3efd/src/org/appwork/utils/os/DesktopSupportLinux.java#L318)). So just setting it in `Dockerfile` to `/opt/JDownloader/Downloads` would fix the issue.

Afterwards, JDownloader checks for [`System.getProperty("user.home")`](https://github.com/svn2github/appwork-utils/blob/41e2dedd169c41478f8f6057c49637af685a3efd/src/org/appwork/utils/os/CrossSystem.java#L378) which is populated by [`/etc/passwd`](https://stackoverflow.com/a/1503500/3779655) and can be set by running [`java -Duser.home=$HOME`](https://stackoverflow.com/a/1503436/3779655).

Only as a last option, [it uses the JDownloader root directory](https://github.com/svn2github/appwork-utils/blob/41e2dedd169c41478f8f6057c49637af685a3efd/src/org/appwork/utils/os/CrossSystem.java#L382).

**Comment by darkdragon-001** [comment]
Please consider merging this commit of the PR though!
